documentation/asciidoc/computers/camera/streaming.adoc

```diff
@@ -1,74 +1,82 @@
 == Stream video over a network with `rpicam-apps`
 
-This section describes native streaming from `rpicam-vid`. You can also use the xref:camera_software.adoc#libav-integration-with-rpicam-vid[`libav`] backend for network streaming.
+This section describes how to stream video over a network using `rpicam-vid`. Whilst it's possible to stream very simple formats without using `libav`, for most applications we recommend using the xref:camera_software.adoc#libav-integration-with-rpicam-vid[`libav` backend].
 
 === UDP
 
 To stream video over UDP using a Raspberry Pi as a server, use the following command, replacing the `<ip-addr>` placeholder with the IP address of the client or multicast address and replacing the `<port>` placeholder with the port you would like to use for streaming:
 
 [source,console]
 ----
-$ rpicam-vid -t 0 --inline -o udp://<ip-addr>:<port>
+$ rpicam-vid -t 0 -n --inline -o udp://<ip-addr>:<port>
 ----
 
 To view video streamed over UDP using a Raspberry Pi as a client, use the following command, replacing the `<port>` placeholder with the port you would like to stream from:
 
 [source,console]
 ----
-$ vlc udp://@:<port> :demux=h264
+$ ffplay udp://@:<port> -fflags nobuffer -flags low_delay -framedrop
 ----
+As noted previously, `vlc` no longer handles unencapsulated h264 streams.
 
-Alternatively, use the following command on a client to stream using `ffplay`:
+In fact, support for unencapsulated h264 can generally be quite poor so it is often better to send an MPEG-2 Transport Stream instead. Making use of `libav`, this can be accomplished with:
```

Comment:
Should the use of "h264" here actually be "H.264"? Or does it not matter?

Reply:
h264 or H264 or h.264 or H.264? I think folks understand they're all the same, but I think H.264 is probably most common, so will use that.